Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -555,6 +555,14 @@ - (void)deleteBackward {
   }
 }
 #else // [macOS
+- (BOOL)performKeyEquivalent:(NSEvent *)event {
+  if (!self.hasMarkedText && ![self.textInputDelegate textInputShouldHandleKeyEvent:event]) {
+    return YES;
+  }
+
+  return [super performKeyEquivalent:event];
+}
```

Comment:
Why do we need this? Was the override in `keyDown` not sufficient? Should we also override `submitOnKeyDownIfNeeded` to account for the new prop `passthroughAllKeyEvents` ?

Reply:
I don't believe the override in `keyDown` is sufficient -- it isn't invoked if `performKeyEquivalent` returns true (e.g. for Cmd+A).

As for `submitOnKeyDownIfNeeded` -- that seems like a completely different piece of functionality (like HTML form "submit", not "submit" events to JS? Or that's what gather from it -- it fires a custom callback (the submit callback) for specific key strokes (and optionally clear the text input).

Reply:
I'm a derp, that method does something else.